src/Indicator/PhpUnitTestCaseIndicator.php

```diff
@@ -48,4 +48,27 @@ public function isPhpUnitClass(Tokens $tokens, $index)
 
         return false;
     }
+
+    /**
+     * @param Tokens $tokens
+     *
+     * @return \Generator| [int,int] array of [start, end] indexes from sooner to later classes
```

Comment:
`\Generator| [int,int]` syntax doesn't look valid

Reply:
if this is the last thing blocking a merge I'm happy to update this PR :)   what should it be, returned is a `generator` that provides `[int, int]`